exercises/C-variables/exercise.js

```diff
@@ -1,3 +1,5 @@
 // Start by creating a variable `greeting`
-
+const greeting = "Hello world!"
```

Comment:
Don't forget the semicolon at the end of the command (it isn't mandatory in JS but it's better to use it)